lms/static/sass/shared/_fields.scss

```diff
@@ -0,0 +1,69 @@
+// lms - shared - fields
+// ====================
+
+
+.u-field {
+    padding: $baseline 0;
+    border-bottom: 1px solid $gray-l5;
+
+    .message-error {
+        color: $alert-color;
+    }
+
+    .message-validation-error {
+        color: $warning-color;
+    }
+
+    .message-in-progress {
+        color: $gray-d2;
+    }
+
+    .message-success {
+        color: $success-color;
+    }
+}
+
+.u-field-readonly {
+    input[type="text"],
+    input[type="text"]:focus {
+        background-color: transparent;
+        padding: 0px;
+        border: none;
+        box-shadow: none;
+    }
+}
+
+.u-field-title {
+    width: flex-grid(3, 12);
+    display: inline-block;
+    color: $dark-gray1;
+    vertical-align: top;
+    margin-bottom: 0;
```

Comment:
@marcotuts Making the field title clickable wasn't requested for a11y reasons. I'll `pointer-events: none` to `.u-field-title`.